test/articlesection.spec.js

```diff
@@ -40,6 +40,60 @@ describe('An Article Section (collapsible)', function () {
     expect(articleSection.$body.classList.contains('visuallyhidden')).to.be.true;
   });
 
+  it('is initially opened if initial state is closed but the fragment is the section', function () {
+    $elm.dataset.initialState = 'closed';
+    let windowMock = {
```

Comment:
can this mock be extracted at the test case level, to be easily created in different tests (e.g. `let windowMock = withHash('#introduction')`)?